test/common/http/codec_impl_fuzz_test.cc

```diff
@@ -537,7 +553,10 @@ void codecFuzz(const test::common::http::CodecImplFuzzTestCase& input, HttpVersi
         client_server_buf_drain();
       }
     }
-    client_server_buf_drain();
+    // Drain all remaining buffers, unless the connection is effectively closed.
+    if (!should_close_connection) {
+      client_server_buf_drain();
```

Comment:
Should DebugMode also be guarded against closed connection drains?

Reply:
:/ Sorry to be really nit-picky but maybe `client_server_buf_drain()`/`client/server_write_buf.drain()` should take the bool? Just thinking it might hang if the drain is the next action.

Reply:
I think we are safe for these actions, since the `for` loop now has a conjunct in its conditional to ensure we don't continue the loop if closed.